sphinx_gallery/scrapers.py

```diff
@@ -415,24 +514,26 @@ def _single_line_sanitize(s):
     *
 """
 
-HLIST_IMAGE_TEMPLATE = """
+HLIST_SG_TEMPLATE = """
     *
 
-      .. image:: /%s
+      .. image-sg:: /%s
           :alt: %s
+          :srcset: %s
           :class: sphx-glr-multi-img
 """
 
-SINGLE_IMAGE = """
-.. image:: /%s
-    :alt: %s
-    :class: sphx-glr-single-img
```

Comment:
Let's keep this constant around since it's likely that custom scrapers in the wild have used it, and the maintenance cost of keeping this in the namespace is negligible.